src/edt/2024.Dockerfile

```
@@ -0,0 +1,94 @@
+
+# Установка переменных окружения для корректной работы локали
+ENV LANG=ru_RU.UTF-8
+ENV LANGUAGE=ru_RU:ru
+ENV LC_ALL=ru_RU.UTF-8
+
+# Copy EDT
+COPY --from=installer /opt/1C/1CE /opt/1C/1CE
+
+ENV PATH="/opt/1C/1CE/components/1c-enterprise-ring:/opt/1C/1CE/components/1cedt:$PATH"
+
+ENTRYPOINT [ "1cedtcli" ]]
```

Hadolint (2.12.0)

[error] unexpected ']'
expecting '\', a new line followed by the next instruction, or at least one space

(DL1000)